### PR TITLE
add maxconn value to haproxy

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/all-in-one/issues/3566

### CODE REVIEW

File: Containers/docker-socket-proxy/haproxy.cfg [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Inspiration: https://github.com/Tecnativa/docker-socket-proxy/blob/master/haproxy.cfg
 
+global
+    maxconn 2000
```
Comment (Zoey2936) [inline Containers/docker-socket-proxy/haproxy.cfg:4]: is there are reason for using this number?

Comment (szaimen) [reply Containers/docker-socket-proxy/haproxy.cfg:4]: No

Comment (Zoey2936) [reply Containers/docker-socket-proxy/haproxy.cfg:4]: do you know how many connection Appapi uses?

Comment (szaimen) [reply Containers/docker-socket-proxy/haproxy.cfg:4]: @andrey18106 @bigcat88 do you know?

Comment (andrey18106) [reply Containers/docker-socket-proxy/haproxy.cfg:4]: Is it that necessary? AppAPI uses connection to daemon only during administration actions

Comment (szaimen) [reply Containers/docker-socket-proxy/haproxy.cfg:4]: Apparently maxconn is needed on some system. See https://github.com/nextcloud/all-in-one/issues/3566

Comment (andrey18106) [reply Containers/docker-socket-proxy/haproxy.cfg:4]: It would be enough with reserve set it to 10 I guess.

Comment (szaimen) [reply Containers/docker-socket-proxy/haproxy.cfg]: all right. adjusted.

**Review by andrey18106 — APPROVED** [review]

**Pull request merged**